src/autorest-configuration.md

```diff
@@ -68,4 +68,19 @@ help-content:
       - key: export-clients
         type: boolean
         description: Indicates if generated clients are to be exported.  Default to true for ARM, false for data-plane.
+      - key: header-text
```

Comment:
Shouldn't these be added to the autorest front-end?  They're not specific to autorest.go

Reply:
Thoughts @timotheeguerin ?

Reply:
I slapped `autorest --go --help` and did not see them.

Reply:
Yeah I don't think the autorest help content contains all the options.

It is here if you want to add missing ones. https://github.com/Azure/autorest/blob/main/packages/libs/configuration/resources/help-configuration.md